stubs/whatthepatch/whatthepatch/snippets.pyi

```diff
@@ -0,0 +1,7 @@
+from collections.abc import Iterable
+from typing import Any
+
+def remove(path: str) -> None: ...
```

Comment:
This function actually works with any `StrOrBytesPath` (defined in `_typeshed`). But this doesn't matter much if the rest of the library only works with string paths, not e.g. bytes paths or `pathlib.Path` objects.

Reply:
Thanks for your detailed code reviews.